lib/collection/src/shards/replica_set.rs

```diff
@@ -675,11 +697,11 @@ impl ShardReplicaSet {
                 match read_operation_res {
                     Ok(_) => return read_operation_res,
                     res @ Err(CollectionError::ServiceError { .. }) => {
-                        log::debug!("Local read op. failed: {:?}", res.as_ref().err());
+                        log::debug!("Local read op. failed: {}", res.as_ref().err().unwrap());
```

Comment:
Nitpick:

```diff
-err().unwrap()
+unwrap_err()
```

Note: there are more occurrences like this in this file.

Reply:
```
`Res` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
```

Reply:
Interesting! Rust weirdness.